Makefile

```diff
@@ -28,18 +28,24 @@ test:
 coverage: test
 	go tool cover -html=coverage.out -o coverage.html
 
-test-macos-interactive-certificate-install:
-	go test -coverprofile=coverage_osx.out -v -test.run=TestSetupOurCert_InteractiveCertificateInstall ./dev
+test-macos-interactive:
+	@echo "This will break your existing puma-dev setup. You'll need to run setup/install again. Cool? Cool."
+	@echo "Also, prepare to provide your system password several times."
+	@read -p "Press [return] to continue..."
+	go test ./... -v -test.run=DarwinInteractive -count=1
 
-test-macos-interactive-dev-setup-install: clean build
+test-macos-manual-setup-install: clean build
```

Comment:
this still feels funky, but given that it's so difficult to test this flow in CI, this allows me to test for regressions rapidly.

Reply:
It's fine.